pages/en/hosted-service/what-is-hosted-service.mdx

```diff
@@ -4,7 +4,7 @@ title: What is the Hosted Service?
 
 This section will walk you through deploying a subgraph to the Hosted Service, otherwise known as the [Hosted Service.](https://thegraph.com/hosted-service/) As a reminder, the Hosted Service will not be shut down soon. We will gradually sunset the Hosted Service once we reach feature parity with the decentralized network. Your subgraphs deployed on the Hosted Service are still available [here.](https://thegraph.com/hosted-service/)
 
-If you don't have an account on the Hosted Service, you can signup with your Github account. Once you authenticate, you can start creating subgraphs through the UI and deploying them from your terminal. Graph Node supports a number of Ethereum testnets (Rinkeby, Ropsten, Kovan) in addition to mainnet.
+If you don't have an account on the Hosted Service, you can signup with your Github account. Once you authenticate, you can start creating subgraphs through the UI and deploying them from your terminal. Graph Node supports several Ethereum testnets (Rinkeby, Ropsten, Kovan) in addition to mainnet.
```

Comment:
I think this sentence understates the number of supported networks at this point, but that is perhaps outside the scope of this PR